From 7571726b1f60cfd72eff819cacbb09c2f70e5598 Mon Sep 17 00:00:00 2001
From: 86183 <86183@DESKTOP-SABKKDC>
Date: Thu, 20 Jun 2019 16:47:12 +0800
Subject: [PATCH] change:update PUMessageBox alpha

---
 Panuon.UI/Themes/Generic.xaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Panuon.UI/Themes/Generic.xaml b/Panuon.UI/Themes/Generic.xaml
index 74aeebb..5ad43e3 100644
--- a/Panuon.UI/Themes/Generic.xaml
+++ b/Panuon.UI/Themes/Generic.xaml
@@ -1233,7 +1233,7 @@
             <Trigger Property="AnimationStyle"
                      Value="Scale">
                 <Setter Property="Opacity"
-                        Value="0.5"></Setter>
+                        Value="0.1"></Setter>
                 <Setter Property="Template">
                     <Setter.Value>
                         <ControlTemplate TargetType="{x:Type local:PUWindow}">
@@ -1499,7 +1499,7 @@
                                             <Storyboard>
                                                 <DoubleAnimation Storyboard.TargetName="bdrCover"
                                                                  Storyboard.TargetProperty="Opacity"
-                                                                 To="0.7"
+                                                                 To="0.2"
                                                                  Duration="0:0:0.2"></DoubleAnimation>
                                                 <BooleanAnimationUsingKeyFrames Storyboard.TargetName="bdrCover"
                                                                                 Storyboard.TargetProperty="IsHitTestVisible">
-- 
2.21.0.windows.1

